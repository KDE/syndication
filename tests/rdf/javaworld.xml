<?xml version="1.0" encoding="UTF-8"?>
<!--
RDF 1.0 with the URIs <items> not matching the actual <item>s
As not matching URIs is a common error, <items> should be ignored and the actual item descriptions should be returned (even if that doesn't make sense on RDF level)
bug 294354
-->



						    <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rss="http://purl.org/rss/1.0/" xmlns:sy="http://purl.org/rss/1.0/modules/syndication/">
       <channel rdf:about="http://www.javaworld.com/features/index.html">
          <title>Featured Articles</title>
          <link>http://www.javaworld.com/features/index.html</link>
          <description>JW RSS feed for Featured Articles</description>
          <dc:publisher>Javaworld</dc:publisher>
          <dc:rights>Copyright(C) 1994 - 2012 Javaworld</dc:rights>
                    <items>
             <rdf:Seq>
															<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120228-modernizing-it.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120224-bjarne-stroustrup-interview.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120222-node-js-tools-remake-server-side.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120216-fatal-exception.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120214-jtip-rss-for-android.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120208-mongodb-review.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120206-programmer-personality-types.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120202-environment-aware-maven-build.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-02-2012/120201-opa-programming.html"/>
							<rdf:li resource="http://www.javaworld.com/javaworld/jw-01-2012/120126-fatal-exception.html"/>
							             </rdf:Seq>
          </items>
       </channel>
											<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120228-modernizing-it.html">
					<title>Electric Cloud CEO: We automate agile development</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120228-modernizing-it.html</link>
					<description>An explosion of tools and platforms is making app dev much more complex. Mike Maciag, CEO of Electric Cloud, offers cross-platform workflow and automation to rationalize it all.</description>
					<dc:creator/>
										<dc:date>2012-02-28T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120224-bjarne-stroustrup-interview.html">
					<title>Stroustrup reveals what&apos;s new in C++ 11</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120224-bjarne-stroustrup-interview.html</link>
					<description>Latest version of language isn&apos;t a major overhaul, but offers many small upgrades to appeal to different areas of development.</description>
					<dc:creator/>
										<dc:date>2012-02-24T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120222-node-js-tools-remake-server-side.html">
					<title>Node.js tools: Server-side JavaScript comes of age</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120222-node-js-tools-remake-server-side.html</link>
					<description>Node&apos;s ecosystem is growing at an exponential rate, which is only natural when you start from scratch. Still, the plethora of Node-inspired tools amount to a diverse and sustainable infrastructure, which could ultimatey redefine the web development stack. </description>
					<dc:creator/>
										<dc:date>2012-02-22T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120216-fatal-exception.html">
					<title>Google Chrome, HTML5, and the new Web platform</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120216-fatal-exception.html</link>
					<description>The Chrome dev team is working toward a vision of Web apps that offers a clean break from traditional websites.</description>
					<dc:creator/>
										<dc:date>2012-02-16T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120214-jtip-rss-for-android.html">
					<title>Java tip: Set up an RSS feed for your Android application</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120214-jtip-rss-for-android.html</link>
					<description>Get a hands-on introduction to using &lt;code&gt;javax.xml.parsers.SAXParser&lt;/code&gt; to parse an RSS feed in XML format. While some familiarity with Java application development is assumed, this Java tip is suitable for developers new to Java mobile development.</description>
					<dc:creator/>
										<dc:date>2012-02-14T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120208-mongodb-review.html">
					<title>Flexing NoSQL: MongoDB in review</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120208-mongodb-review.html</link>
					<description>MongoDB shines with broad programming language support, SQL-like queries, and out-of-the-box scaling. Check the InfoWorld Review score card to see how MongoDB stacks up among NoSQL data stores.</description>
					<dc:creator/>
										<dc:date>2012-02-08T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120206-programmer-personality-types.html">
					<title>Programmer personality types: 13 profiles in code</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120206-programmer-personality-types.html</link>
					<description>From Underdocumenters to flat-out Fakers, the world of software development is full of characters. Here, InfoWorld&apos;s Peter Wayner offers 13 developer personality profiles based on coding style.</description>
					<dc:creator/>
										<dc:date>2012-02-06T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120202-environment-aware-maven-build.html">
					<title>Develop an environment-aware Maven build process</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120202-environment-aware-maven-build.html</link>
					<description>Including environment variables in your Maven build process could boost your team&apos;s  efficiency at every stage of the software development lifecycle. Java developer Paul Spinelli demonstrates his custom approach to environment-aware Maven builds.</description>
					<dc:creator/>
										<dc:date>2012-02-02T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-02-2012/120201-opa-programming.html">
					<title>Programming Opa: Web development, reimagined</title>
					<link>http://www.javaworld.com/javaworld/jw-02-2012/120201-opa-programming.html</link>
					<description>MLstate&apos;s Opa streamlines Web app development with a single language for client and server, but the bright promise is not without pitfalls.</description>
					<dc:creator/>
										<dc:date>2012-02-01T00:00:00Z</dc:date>
				</item>
							<item rdf:about="http://www.javaworld.com//javaworld/jw-01-2012/120126-fatal-exception.html">
					<title>Here&apos;s how to solve America&apos;s developer shortage</title>
					<link>http://www.javaworld.com/javaworld/jw-01-2012/120126-fatal-exception.html</link>
					<description>Employers say they can&apos;t find enough workers to staff IT jobs. Maybe they&apos;ve been going about it all wrong.</description>
					<dc:creator/>
										<dc:date>2012-01-26T00:00:00Z</dc:date>
				</item>
							    </rdf:RDF>
