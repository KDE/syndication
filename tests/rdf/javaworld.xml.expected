# Feed begin ######################
title: #Featured Articles#
link: #http://www.javaworld.com/features/index.html#
description: #JW RSS feed for Featured Articles#
copyright: #Copyright(C) 1994 - 2012 Javaworld#
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120201-opa-programming.html#
title: #Programming Opa: Web development, reimagined#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120201-opa-programming.html#
description: #MLstate's Opa streamlines Web app development with a single language for client and server, but the bright promise is not without pitfalls.#
datePublished: #Wed Feb 1 00:00:00 2012#
dateUpdated: #Wed Feb 1 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120222-node-js-tools-remake-server-side.html#
title: #Node.js tools: Server-side JavaScript comes of age#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120222-node-js-tools-remake-server-side.html#
description: #Node's ecosystem is growing at an exponential rate, which is only natural when you start from scratch. Still, the plethora of Node-inspired tools amount to a diverse and sustainable infrastructure, which could ultimatey redefine the web development stack.#
datePublished: #Wed Feb 22 00:00:00 2012#
dateUpdated: #Wed Feb 22 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120216-fatal-exception.html#
title: #Google Chrome, HTML5, and the new Web platform#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120216-fatal-exception.html#
description: #The Chrome dev team is working toward a vision of Web apps that offers a clean break from traditional websites.#
datePublished: #Thu Feb 16 00:00:00 2012#
dateUpdated: #Thu Feb 16 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120208-mongodb-review.html#
title: #Flexing NoSQL: MongoDB in review#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120208-mongodb-review.html#
description: #MongoDB shines with broad programming language support, SQL-like queries, and out-of-the-box scaling. Check the InfoWorld Review score card to see how MongoDB stacks up among NoSQL data stores.#
datePublished: #Wed Feb 8 00:00:00 2012#
dateUpdated: #Wed Feb 8 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120224-bjarne-stroustrup-interview.html#
title: #Stroustrup reveals what&apos;s new in C++ 11#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120224-bjarne-stroustrup-interview.html#
description: #Latest version of language isn't a major overhaul, but offers many small upgrades to appeal to different areas of development.#
datePublished: #Fri Feb 24 00:00:00 2012#
dateUpdated: #Fri Feb 24 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120206-programmer-personality-types.html#
title: #Programmer personality types: 13 profiles in code#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120206-programmer-personality-types.html#
description: #From Underdocumenters to flat-out Fakers, the world of software development is full of characters. Here, InfoWorld's Peter Wayner offers 13 developer personality profiles based on coding style.#
datePublished: #Mon Feb 6 00:00:00 2012#
dateUpdated: #Mon Feb 6 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120202-environment-aware-maven-build.html#
title: #Develop an environment-aware Maven build process#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120202-environment-aware-maven-build.html#
description: #Including environment variables in your Maven build process could boost your team's  efficiency at every stage of the software development lifecycle. Java developer Paul Spinelli demonstrates his custom approach to environment-aware Maven builds.#
datePublished: #Thu Feb 2 00:00:00 2012#
dateUpdated: #Thu Feb 2 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120228-modernizing-it.html#
title: #Electric Cloud CEO: We automate agile development#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120228-modernizing-it.html#
description: #An explosion of tools and platforms is making app dev much more complex. Mike Maciag, CEO of Electric Cloud, offers cross-platform workflow and automation to rationalize it all.#
datePublished: #Tue Feb 28 00:00:00 2012#
dateUpdated: #Tue Feb 28 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-02-2012/120214-jtip-rss-for-android.html#
title: #Java tip: Set up an RSS feed for your Android application#
link: #http://www.javaworld.com/javaworld/jw-02-2012/120214-jtip-rss-for-android.html#
description: #Get a hands-on introduction to using <code>javax.xml.parsers.SAXParser</code> to parse an RSS feed in XML format. While some familiarity with Java application development is assumed, this Java tip is suitable for developers new to Java mobile development.#
datePublished: #Tue Feb 14 00:00:00 2012#
dateUpdated: #Tue Feb 14 00:00:00 2012#
# Item end ########################
# Item begin ######################
id: #http://www.javaworld.com//javaworld/jw-01-2012/120126-fatal-exception.html#
title: #Here&apos;s how to solve America&apos;s developer shortage#
link: #http://www.javaworld.com/javaworld/jw-01-2012/120126-fatal-exception.html#
description: #Employers say they can't find enough workers to staff IT jobs. Maybe they've been going about it all wrong.#
datePublished: #Thu Jan 26 00:00:00 2012#
dateUpdated: #Thu Jan 26 00:00:00 2012#
# Item end ########################
# Feed end ########################

